bin/run_tests.sh

```diff
@@ -4,6 +4,13 @@ set -e
 base=$(pwd)
 failed=0
 
+# run snyk tests locally. Skip in bamboo due to authentication token requirement.
+echo "Scanning Infrastructure as code with Snyk for potential vulnerabilities..."
+snyk iac test || true
+echo "Scanning open-source code with Snyk for potential vulnerabilities..."
+cd website && snyk test -d || true
+cd -
+
```

Comment:
Need to research and come up with a way to scan our python requirements. I would recommend doing the check at each individual lambda level per our conversation. Depending on the situation you may either want to recommend a seperate script for each lambda or integrating it into run_test script for each lambda.

Note that the code will also need to be added to the graphql code as well at some point.

Reply:
After some research with snyk for scanning requirements.txt files, it is giving the following error on mac even after trying all solutions available on the web.
```
copy_from_archive rhassan$ snyk test --file=requirements.txt --package-manager=pip --command=python3
Failed to test pip project 
```
Solutions tried: 
https://stackoverflow.com/questions/67110095/snyk-test-returns-failed-to-test-pip-project-error

https://support.snyk.io/hc/en-us/articles/360007419257-Required-packages-missing-when-testing-a-Python-project-with-pip

https://github.com/snyk/cli/issues/730

https://www.freecodecamp.org/news/how-to-find-and-fix-security-vulnerabilities-using-snyk/

Reply:
created https://bugs.earthdata.nasa.gov/browse/ORCA-598
`Add snyk scan for all lambda requirements.txt and requirements-dev.txt files`